### DISCUSSION TITLE
Keine Daten

### DISCUSSION THREAD

**Discussion by heckmic** [syssi/esphome-smg-ii#39] [General]
Hallo, 

habe 2 SMG II hier, die mit  SmartESS ausgelesen werden können. Ich habe alles verdrahtet und bereits 3 MAX323 und 3 D1 minis getestet. Kabel passen.
Leider bekomme ich bis auf:

INFO ESPHome 2023.8.2
INFO Reading configuration /opt/iobroker/iobroker-data/esphome.0/mapawr1.yaml...
INFO Detected timezone 'Europe/Berlin'
INFO Starting log output from mapawr1/debug
INFO Connected to MQTT broker!

gar nichts weiter zurück.

**Comment by syssi** [comment]
Kannst du bitte das Log-Level auf `DEBUG` anheben und den `mqtt:`-Abschnitt vorerst aus deiner YAML entfernen? Was siehst du dann im Log? Kannst du ein Foto deines Verkabelung machen? Wichtig ist, dass man erkennt, welche GPIOs du nutzt und wie die einzelnen Komponenten verdrahtet sind.

**Reply by heckmic** [reply]
Danke für deine Antwort! Ich habe den ESP8266 gegen einen ESP32 getauscht. Irgendwas ist da mit dem TX/RX, da passiert gar nichts. Die nächste Frage die sich mir stellt ist aber, wie schreibe ich via mqtt die einstellungen? Trage ich bei output-priority einfach eine 1 oder 2 ein, passiert gar nichts.

**Reply by syssi** [reply]
Wenn der ESP startet schreibt er einmal alle Topics ins Logbuch. Hier findest du auch die Topics, auf welchen man die neuen Zustände publizieren muss. Sie unterscheiden sich vom Topic, auf welchem der aktuelle Zustand zu finden ist.
